src/components/People.tsx

```diff
@@ -0,0 +1,131 @@
+import { Loader } from './Loader';
+import { Link, useParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { useEffect, useState } from 'react';
+import { Person } from '../types';
+import classNames from 'classnames';
+import { PersonLink } from './PersonLink';
+
+export const People = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [errorMessage, setErrorMessage] = useState('');
+  const { personSlug } = useParams();
+
+  useEffect(() => {
+    getPeople()
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
  const [errorMessage, setErrorMessage] = useState('');
  const { personSlug } = useParams();

  useEffect(() => {
    setIsLoading(true);
    
    getPeople()
```
You should set the loader when you start loading something and remove it after loading is finished